doc/en/example/simple.rst

```diff
@@ -655,12 +657,12 @@ and run it::
 
     $ py.test -s test_module.py
     ======= test session starts ========
-    platform linux2 -- Python 2.7.9, pytest-2.8.0.dev4, py-1.4.28, pluggy-0.3.0
+    platform linux -- Python 3.4.2, pytest-2.8.1.dev1, py-1.4.30, pluggy-0.3.1
     rootdir: $REGENDOC_TMPDIR, inifile: 
     collected 3 items
     
-    test_module.py E('setting up a test failed!', 'test_module.py::test_setup_fails')
-    F('executing test failed', 'test_module.py::test_call_fails')
+    test_module.py Esetting up a test failed! test_module.py::test_setup_fails
+    Fexecuting test failed test_module.py::test_call_fails
```

Comment:
This probably intends to print a tuple but doesn't anymore with Python 3 - so I think this should be changed to `print(("...", ...))` in the example above.